<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>average_window</class>
 <widget class="QWidget" name="average_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Define Average Window - ${DEV}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="3" column="1">
      <widget class="QLineEdit" name="start_line_edit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Enter start value (us)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Write to PV: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="PyDMLabel" name="get_pv_size_i">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${DEV}:I${R}${N}.NELM</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="stop_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>End (us)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="start_line_edit_q">
       <property name="placeholderText">
        <string>Enter start value (us)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="3">
      <layout class="QHBoxLayout" name="button_layout"/>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Size</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="start_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Start (us)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="end_line_edit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Enter end value (us)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="PyDMLabel" name="get_pv_size_q">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${DEV}:Q${R}${N}.NELM</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="end_line_edit_q">
       <property name="placeholderText">
        <string>Enter end value (us)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Q Window</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="5">
      <widget class="PyDMWaveformPlot" name="average_window_wf">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>300</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="showXGrid">
        <bool>true</bool>
       </property>
       <property name="showYGrid">
        <bool>true</bool>
       </property>
       <property name="axisColor">
        <color>
         <red>204</red>
         <green>204</green>
         <blue>204</blue>
        </color>
       </property>
       <property name="showRightAxis">
        <bool>true</bool>
       </property>
       <property name="title" stdset="0">
        <string>I/Q windows preview</string>
       </property>
       <property name="xLabels">
        <stringlist>
         <string>Time (us)</string>
        </stringlist>
       </property>
       <property name="yLabels">
        <stringlist>
         <string>Amplitude</string>
        </stringlist>
       </property>
       <property name="showLegend">
        <bool>true</bool>
       </property>
       <property name="curves">
        <stringlist>
         <string>{&quot;y_channel&quot;: null, &quot;x_channel&quot;: &quot;ca://${DEV}:XWF&quot;, &quot;name&quot;: &quot;I&quot;, &quot;color&quot;: &quot;#ff557f&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;redraw_mode&quot;: 2}</string>
         <string>{&quot;y_channel&quot;: null, &quot;x_channel&quot;: &quot;ca://${DEV}:XWF&quot;, &quot;name&quot;: &quot;Q&quot;, &quot;color&quot;: &quot;#5500ff&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;redraw_mode&quot;: 2}</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>I Window</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="4" rowspan="2">
      <widget class="QPushButton" name="draw_window_pb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Draw Windows</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="display_pv_label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{color: #034b61}</string>
       </property>
       <property name="text">
        <string>${TITLE}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMWaveformPlot</class>
   <extends>QGraphicsView</extends>
   <header>pydm.widgets.waveformplot</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
